http-api/src/main/java/net/runelite/http/api/gson/InstantTypeAdapter.java

```diff
@@ -43,12 +45,8 @@ public void write(JsonWriter out, Instant value) throws IOException
 			return;
 		}
 
-		out.beginObject()
-			.name("seconds")
-			.value(value.getEpochSecond())
-			.name("nanos")
-			.value(value.getNano())
-			.endObject();
+		long jsTime = (value.getEpochSecond() * 1000) + (value.getNano() / NS_PER_MS);
```

Comment:
You can also use `value.getLong(ChronoField.MILLI_OF_SECOND)` for getting the ms I think.